src/pages/coaches/will-voelcker.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Will Voelcker
+title: Senior Full-stack JavaScript Engineer
+companyName: LeanJS
+companyLink: https://leanjs.com
+gitHub: https://github.com/wvoelcker
+linkedIn: https://uk.linkedin.com/in/willv
+blockquote: I love working with developers to help them improve their skills, stretch their brains, further their understanding, and advance their career.
+imageDescription: React GraphQL Academy coach Will Voelcker, smiling with coach Alex in the background
+imageSrc: https://firebasestorage.googleapis.com/v0/b/reactjsacademy-react.appspot.com/o/team%2Fwill.jpg?alt=media
+tags: []
+---
+
+<ul>
```

Comment:
@ric9176 why don't we use markdown for the list? https://www.markdownguide.org/basic-syntax/#unordered-lists